fdbclient/NativeAPI.actor.cpp

```diff
@@ -7493,6 +7514,7 @@ ACTOR Future<Standalone<VectorRef<KeyRangeRef>>> getBlobGranuleRangesActor(Trans
 		fmt::print("Getting Blob Granules for [{0} - {1})\n", keyRange.begin.printable(), keyRange.end.printable());
 	}
 	self->setOption(FDBTransactionOptions::ACCESS_SYSTEM_KEYS);
+	// FIXME: limit to tenant range if set
```

Comment:
Was this a note to fix before merging or fix at some later date?

Reply:
A note to fix, but could be a separate PR, since none of the downstream work depends on the getRanges or purge calls.